scripts/van_allen_tools/migrate_van_allen_workspaces.py

```diff
@@ -347,6 +347,24 @@ def copy_workspace_entities(destination_workspace_namespace, destination_workspa
     return True, data_table_name_list
 
 
+def get_workspace_attributes(source_workspace_namespace, source_workspace_name):
```

Comment:
Since this function is pretty general, and can be re-used to get workspace attributes from any workspace, we could put this into the `utils.py` and just import it into this larger one! The only thing to change, and this is like a tiny minor details, would be the function variables to `project` and `workspace` - or whatever the other functions in `utils.py` use to denote a terra project and workspace name. It should be in the same directory as where this script is